### DISCUSSION TITLE
CrossTestDiscussion

### DISCUSSION THREAD

**Discussion by nihalchari** [eclipse-iceoryx/iceoryx#551] [Ideas]
-  to discuss about ongoing testing related activities, which are quite a lot and are also having quite some people involved

of course, no disclosure of future planning, strategies....etc. only currently ongoing open items

**Comment by nihalchari** [comment]
So as discussed in gitter private room, i am starting discussion here for Testing on Iceoryx in open

**Reply by CarolinaGGB** [reply]
private Gitter room now deleted

**Comment by JimmyBch** [comment]
Hello,

Should we create a new Git Hub issue for unit test in iceoryx_utils folder ?

**Reply by elfenpiff** [reply]
@JimmyBch I would very welcome it! And please create a ToDo list in there and add the developer who is currently working on it. Like this
- [ ] cxx/SomeClass (@elfenpiff) 
- [ ] cxx/AnotherClass (@JimmyBch)

**Reply by JimmyBch** [reply]
Will be done :)
